### PR TITLE
Implement vendor camera functionality

### PR DESCRIPTION
**Pull request by esanmiguelc**
- Wraps force and cancel methods from NBiometricClient
  - setupCamera creates a camera capture thread for a device's
    front-facing camera

### CODE REVIEW

**Comment by esanmiguelc** [comment]
@KevinLiddle @rmulhol

File: app/src/main/java/co/blustor/gatekeeper/biometrics/GKFaces.java [hunk 1]
```diff
@@ -77,6 +90,96 @@ public Template createTemplateFromStream(InputStream inputStream) throws IOExcep
...
+    public interface OnCameraCompletionListener {
+        Void onSuccess(Template template);
+        Void onFailure();
+    }
+
+    private void setFrontFaceCamera() {
+        for (NDevice device : mBiometricClient.getDeviceManager().getDevices()) {
+            if (device.getDeviceType().contains(NDeviceType.CAMERA)) {
+                if (device.getDisplayName().contains("Front")) {
+                    if (!mBiometricClient.getFaceCaptureDevice().equals(device))
```
Comment (rmulhol) [inline app/src/main/java/co/blustor/gatekeeper/biometrics/GKFaces.java]: Maybe extract these conditions into a separate function and combine them (`&&`)?

**Comment by rmulhol** [comment]
Only one small suggestion, otherwise :+1:

**Comment by KevinLiddle** [comment]
:+1:

**Pull request merged**